daq_macros.py

```diff
@@ -1847,7 +1848,9 @@ def snakeRasterBluesky(rasterReqID, grain=""):
         xMotAbsoluteMove, xEnd, yMotAbsoluteMove, yEnd, zMotAbsoluteMove, zEnd = raster_positions(row, stepsize, omegaRad, rasterStartX, rasterStartY, rasterStartZ, row_index)
         vector = {'x': (xMotAbsoluteMove, xEnd), 'y': (yMotAbsoluteMove, yEnd), 'z': (zMotAbsoluteMove, zEnd)}
         yield from zebraDaqRasterBluesky(raster_flyer, omega, numsteps, img_width_per_cell * numsteps, img_width_per_cell, exptimePerCell, rasterFilePrefix,
-            data_directory_name, file_number_start, vector)
+            data_directory_name, file_number_start, row_index, vector)
+        raster_flyer.zebra.reset.put(1)  # reset after every row to make sure it is clear for the next row
+        time.sleep(0.2)
```

Comment:
A comment about the reason for sleep would be helpful

Reply:
it's in the comment for this commit, but it should be highlighted in the line. thanks for the reminder. will resolve when I add a comment

Reply:
comment added in https://github.com/JunAishima/lsdc/commit/f61549d6351290cb52632e3be8aa3071c5f3db96